src/main/java/library/app/spring/controller/BookController.java

```diff
@@ -0,0 +1,30 @@
+package library.app.spring.controller;
+
+import library.app.spring.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.ModelMap;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+@RequestMapping("/book")
+public class BookController {
+
+    @Autowired
+    private BookService bookService;
+
+    @GetMapping("/all")
```

Comment:
what about other methods?
for example: findByTitle, add

Reply:
done